frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_file_exporter_widget.dart

```diff
@@ -182,3 +209,30 @@ extension AppFlowy on dartz.Either {
     return null;
   }
 }
+
+class _AppFlowyFileExporter {
+  static Future<void> exportToPath(String path, List<ViewPB> views) async {
+    final documentService = DocumentService();
+    for (final view in views) {
+      String? content;
+      String? extension;
+      switch (view.layout) {
+        case ViewLayoutTypePB.Document:
+          final document = await documentService.openDocument(view: view);
+          document.fold(
+            (l) => content = l.content,
+            (r) => Log.error(r),
+          );
+          extension = 'afdoc';
+          break;
+        default:
+          // todo: @nathan, support other layout types
```

Comment:
I'd use GitHub username here. `insert_page_command` has the following format for my TODOs
```
TODO(a-wallen): Show error dialog here.
```

Reply:
```suggestion
          // TODO(appflowy), support other layout types
```

Reply:
Nice, we can follow this style then.